packages/strapi-helper-plugin/lib/src/components/Input/styles.scss

```diff
@@ -221,3 +254,65 @@
 .spacer {
   height: .5rem;
 }
+
+.inputToggle {
+  margin-bottom: 1.5rem;
+  font-size: 1.3rem;
+
+}
+
+.inputToggleButtons {
+  padding-top: 9px;
+  > button {
```

Comment:
Please sort the properties in a better order. Block properties first (width, height, display, margin, padding, border), then text properties (line-height, color, font) and finally extra properties (transition, -webkit-,  etc)